breakpad-symbols/src/lib.rs

```diff
@@ -27,16 +29,16 @@
 //!     let symbolizer = Symbolizer::new(supplier);
 //!
 //!     // Simple function name lookup with debug file, debug id, address.
-//!     assert_eq!(symbolizer.get_symbol_at_address("test_app.pdb",
-//!         "5A9832E5287241C1838ED98914E9B7FF1",
-//!         0x1010)
+//! let debug_id = DebugId::from_str("5A9832E5287241C1838ED98914E9B7FF1").unwrap();
```

Comment:
maybe rustfmt the doc code blocks as well, like so: https://github.com/getsentry/sentry-rust/blob/master/rustfmt.toml

But that will probably introduce a ton of unrelated changes, so maybe just correctly indent this one ;-)